### PR TITLE
Updating SDK to use Branch Key.

### PR DESCRIPTION
**Pull request by shortstuffsushi**
@aaustin @Sarkar 

Updating SDK with Branch Key.
App Key will continue to work for the time being.
Updating BranchRequest to always send key.

### CODE REVIEW

**Comment by shortstuffsushi** [comment]
I'll try to fix the formatting. Not sure why this was started using tabs :disappointed:

**Comment by shortstuffsushi** [comment]
Ok, formatting is fixed. This is ready to go w/ review.

File: Branch-Xamarin-SDK/Branch.cs [hunk 1]
```diff
@@ -40,11 +40,11 @@ public class Branch
...
 
 		/// <summary>
-		/// Gets or sets the application key. This is generally
+		/// Gets or sets the Branch key. This is generally
 		/// set by the platform specific Init methods.
 		/// </summary>
-		/// <value>The app key.</value>
-		public String AppKey { get; set; }
+		/// <value>The Branch key.</value>
+		public string BranchKey { get; set; }
```
Comment (Sarkar) [inline Branch-Xamarin-SDK/Branch.cs]: in case anyone was wondering, in c# `string` and `String` mean the same - thanks @shortstuffsushi

Comment (shortstuffsushi) [inline Branch-Xamarin-SDK/Branch.cs]: Shoot, I thought I reverted that :disappointed: Was trying to avoid any cosmetic changes with this PR.

**Comment by aaustin** [comment]
Did Sahil give this his blessing?

**Comment by aaustin** [comment]
Can you update the Readme with instructions on using the Branch key instead of the app id?

**Comment by Sarkar** [comment]
code wise yes, haven't tested it locally but i don't assume anything is
wrong with it

On Wed, May 20, 2015 at 3:18 PM, Alex Austin <notifications@github.com>
wrote:

> Did Sahil give this his blessing?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/BranchMetrics/Branch-Xamarin-SDK/pull/4#issuecomment-104059517>
> .
>



-- 

*Sahil Verma*

*Branch Metrics *

w branch.io <https://branch.io/?bmp=sv> | c 510-386-1750 | s sahilroyverma


*Find the Branch team at these upcoming events:*
May 21 - Mobile Growth Hackers
<http://www.meetup.com/Seattle-Mobile-Growth-Hackers-Meetup/events/222295507/>
-
Seattle
May 27 - Mobile Growth Hackers
<http://www.meetup.com/Bay-Area-Mobile-Growth-Hackers/events/222559906/> -
San Francisco
June 11 - GMIC <http://nyc.thegmic.com/> - New York City

**Comment by shortstuffsushi** [comment]
I was finally able to get this working on an Android device. Works fine, so I'm gonna go ahead and merge.

**Pull request merged**